src/test/Makefile-client.am

```diff
@@ -234,7 +235,7 @@ ceph_test_rados_api_aio_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_test_rados_api_aio
 
 ceph_test_rados_api_list_SOURCES = test/librados/list.cc
-ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD)
+ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD) $(CEPH_GLOBAL)
```

Comment:
On 6-5-2016 05:18, Kefu Chai wrote:
> In src/test/Makefile-client.am
> <https://github.com/ceph/ceph/pull/8298#discussion_r62284821>:
>
>> @@ -234,7 +235,7 @@ ceph_test_rados_api_aio_CXXFLAGS = $(UNITTEST_CXXFLAGS)
>>  bin_DEBUGPROGRAMS += ceph_test_rados_api_aio
>>
>>  ceph_test_rados_api_list_SOURCES = test/librados/list.cc
>> -ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD)
>> +ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD) $(CEPH_GLOBAL)
>
> @wjwithagen <https://github.com/wjwithagen> this does sound suspicious.
> as librados client should not reference ldap routines. could you post
> the error messages when linking this test executable?

Eeergh, I'll have to rebuild.
Killing the current one, and will remove this addition, we'll see what 
happens